src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -78,7 +79,18 @@ trait BackendCall {
    * expression `read_lines(my_file_var)` would have to call lookupFunction()("my_file_var")
    * during expression evaluation
    */
-  def lookupFunction: String => WdlValue
+  def lookupFunction: String => WdlValue = {
+    def resolveParameter(name: String) = Try(locallyQualifiedInputs.getOrElse(name, throw new Throwable(s"variable $name not found")))
```

Comment:
Nitpick, I know scala doesn't care as much and let's all exceptions, runtime and checked, fly. Still my _java-sense_ tingles when I see `new Throwable`. Preferably, there should be a more descriptive exception class, but as a stopgap, even `new Exception` is better IMHO. It keeps all of our code away from ever interacting with `Exception`'s JVM halting cousin the `Error`.

All that said, I'm _ok_ if these three instances stay as is, but if I touch these lines in the future, I'll be changing them to `Exception`.